### PR TITLE
Added tbench test that produces only the TCP and process load.

### PR DESCRIPTION
**Pull request by poojabs7**
Signed-off-by: poojabs <pooja@oc2013156016.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
@poojabs7 please rebase as make script as pylint compliant , please import 
`from avocado import main `
it will help to pass travis

File: perf/tbench.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import re
+from avocado import Test
+from avocado import main
+from avocado.utils import archive
+from avocado.utils import process
+from avocado.utils import build
+from avocado.utils.software_manager import SoftwareManager
+
+
+class tbench(Test):
```
Comment (abdhaleegit) [inline perf/tbench.py]: Looks good if we have a doc string that explains the testcase, for class and also for each method

File: perf/tbench.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            # Standard output is verbose and merely makes our debug logs huge
+            # so we don't retain it.  It gets parsed for the results.
+            self.results = process.system_output(cmd, shell=True)
+            os.kill(pid, signal.SIGTERM)    # clean up the server
+        else:                           # child
+            server = self.srcdir + '/tbench_srv'
+            os.execlp(server, server)
+        pattern = re.compile(r"Throughput (.*?) MB/sec (.*?) procs")
+        (throughput, procs) = pattern.findall(self.results)[0]
+        self.log.info({'throughput': throughput, 'procs': procs})
```
Comment (abdhaleegit) [inline perf/tbench.py]: a standard practice to include at the end
if __name__ == "__main__":
    main()

**Comment by poojabs7** [comment]
Opened new PR
https://travis-ci.org/avocado-framework-tests/avocado-misc-tests/builds/174249231

**Pull request closed without merging**